### PR TITLE
Fix trusted modules cache creation

### PR DESCRIPTION
**Pull request by jocel1**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | CACHE_FILE_DEFAULT_COUNTRY_MODULES_LIST was not properly used to generate the trusted / untrusted cache list, slowing down a lot each call to getModulesOnDisk()
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | run tests

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10153)

### CODE REVIEW

File: tests/Integration/Core/Module/ModuleCacheTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        parent::setUp();
+    }
+
+    public function testCacheBehaviour()
+    {
+        Module::deleteTrustedXmlCache();
+        Module::getModulesOnDisk();
+        $trustedFileCreationTime = filemtime(_PS_ROOT_DIR_.'/config/xml/trusted_modules_list.xml');
+        sleep(1);
```
Comment (mickaelandrieu) [inline tests/Integration/Core/Module/ModuleCacheTest.php:44]: are you sure you need a `sleep()` here?

**Review by mickaelandrieu — COMMENTED** [review]
ping @Quetzacoalt91 for presta trust stuff :)

File: tests/Integration/Core/Module/ModuleCacheTest.php (hunk 1 above)
Comment (jocel1) [reply tests/Integration/Core/Module/ModuleCacheTest.php:44]: unfortunately yes, I checked my test was failing without the fix, and it without the sleep(1) it was failing randomly

**Comment by Quetzacoalt91** [comment]
@mickaelandrieu this PR is not related to PrestaTrust. :)

It is related to another system, which checks that a module is part of the marketplace database.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
@Quetzacoalt91 aww, I'm not aware at all of this part of PrestaShop ^^

Thanks for review and good job @jocel1

**Pull request merged**